client/js/components/statement/StatementExportModal.vue

```diff
@@ -40,10 +40,45 @@
             :value="key"
             :checked="active === key"
             @change="active = key" />
+          <template v-if="active !== 'xlsx_normal'">
```

Comment:
I think the spacing in the export modal still needs some fixing - the "Art des Exports" label is too close to the heading, and between the checkboxes and the next label there seems to be a lot of empty space, and then also between the inputs and the buttons. Maybe it was like this before though, and we can also treat it as a UI bug.

![modal_spacing](https://github.com/user-attachments/assets/cf2e9d92-6981-432f-a494-a91685f9387c)

Another thing that bothers me is the "zip" label. If it's not too complicated, could you change it to "DOCX als ZIP"? I know it's not part of the story so feel free to ignore it, I just find the "zip" label confusing.